.github/CODEOWNERS

```diff
@@ -364,9 +364,9 @@ packages/react-components/react-icons-compat/stories @microsoft/cxe-red @tomi-ms
 packages/react-components/react-tag-picker @microsoft/teams-prg
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32883&commitId=2295b05e74e28475bccc0a8a4f5c84cbf07fdb65&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.badgeMask.chromium.png|18| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2513| Changed|
|Drawer.overlay drawer full.chromium.png|1167| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>